test/mlir/krnl/krnl_getref_lowering.mlir

```diff
@@ -49,15 +47,14 @@ func @test_getref_lowering_dynamic(%arg0: memref<2x2xf32>) -> memref<2x?xf32> {
   // CHECK: %[[C13_I64:.+]] = llvm.mlir.constant(13 : i64) : !llvm.i64
   // CHECK: %[[C5_INDEX:.+]] = llvm.mlir.constant(5 : index) : !llvm.i64
   // CHECK: %[[C10_INDEX:.+]] = llvm.mlir.constant(10 : index) : !llvm.i64
-  // CHECK: [[MUL1:%.+]] = llvm.mul %[[C10_INDEX]], %[[C5_INDEX]] : !llvm.i64
-  // CHECK: llvm.mlir.null
-  // CHECK: llvm.mlir.constant(1 : index)
-  // CHECK: llvm.getelementptr
-  // CHECK: [[TYPE_SIZE_IN_BYTES:%.+]] = llvm.ptrtoint
+  // CHECK: %[[C1_INDEX:.+]] = llvm.mlir.constant(1 : index) : !llvm.i64
+  // CHECK: %[[MUL1:.+]] = llvm.mul %[[C5_INDEX]], %[[C10_INDEX]] : !llvm.i64
+  // CHECK: [[FLOAT_STAR:%.+]] = llvm.mlir.null : !llvm.ptr<float>
+  // CHECK: [[ELEM1:%.+]] = llvm.getelementptr [[FLOAT_STAR]][%[[MUL1]]] : (!llvm.ptr<float>, !llvm.i64) -> !llvm.ptr<float>
+  // CHECK: [[ELEM_SIZE:%.+]] = llvm.ptrtoint [[ELEM1]] : !llvm.ptr<float> to !llvm.i64
 
   /// Allocate the memory pool alloc.
-  // CHECK: [[ALLOC_SIZE:%.+]] = llvm.mul [[MUL1]], [[TYPE_SIZE_IN_BYTES]] : !llvm.i64
-  // CHECK: [[ALLOC:%.+]] = llvm.call @malloc([[ALLOC_SIZE]]) : (!llvm.i64) -> !llvm.ptr<i8>
+  // CHECK: [[ALLOC:%.+]] = llvm.call @malloc([[ELEM_SIZE]]) : (!llvm.i64) -> !llvm.ptr<i8>
```

Comment:
Again it looks like the multiplication of the element size by MUL1 has been skipped.